Buildings/Experimental/DHC/Networks/Examples/Distribution2PipeExample.mo

```diff
@@ -0,0 +1,283 @@
+    mCon_flow_nominal=mCon_flow_nominal,
+    mEnd_flow_nominal=m_flow_nominal,
+    lDis=fill(6, nLoa),
+    lEnd=1) annotation (Placement(transformation(extent={{40,20},{80,40}})));
+  Buildings.Fluid.Movers.FlowControlled_dp pumAutoSize(
+    redeclare package Medium = Medium1,
+    per(final motorCooledByFluid=false),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
+    m_flow_nominal=m_flow_nominal,
+    addPowerToMedium=false,
+    nominalValuesDefineDefaultPressureCurve=true,
+    use_inputFilter=false,
+    dp_nominal=dp_nominal)
+    annotation (Placement(transformation(extent={{0,20},{20,40}})));
+  Buildings.Fluid.MixingVolumes.MixingVolume volAutoSize(
+    final prescribedHeatFlowRate=true,
+    redeclare final package Medium = Medium1,
+    V=m_flow_nominal*tau/rho_default,
+    final mSenFac=1,
+    final m_flow_nominal=m_flow_nominal,
+    final energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    nPorts=2) "Volume for fluid stream"
+    annotation (Placement(transformation(extent={{-31,20},{-11,40}})));
+  Buildings.Controls.OBC.CDL.Reals.Sources.Constant dpPum(
+    k=dp_nominal)
+    "Prescribed head"
+    annotation (Placement(transformation(extent={{-100,30},{-80,50}})));
+  Buildings.Controls.OBC.CDL.Reals.Sources.Constant THeaWatSup(k=max(
+        terAutoSize.T_aHeaWat_nominal)) "Supply temperature"
+    annotation (Placement(transformation(extent={{-100,0},{-80,20}})));
+  Buildings.Experimental.DHC.Loads.BaseClasses.Validation.BaseClasses.FanCoil2PipeHeatingValve terPlugFlow[nLoa](
+    each final facMul=facMul,
+    redeclare each final package Medium1 = Medium1,
+    redeclare each final package Medium2 = Medium2,
+    each final QHea_flow_nominal=QHea_flow_nominal,
+    each final mLoaHea_flow_nominal=mLoaHea_flow_nominal,
+    each final T_aHeaWat_nominal=T_aHeaWat_nominal,
+    each final T_bHeaWat_nominal=T_bHeaWat_nominal,
+    each final T_aLoaHea_nominal=T_aLoaHea_nominal) "Heating terminal unit"
+    annotation (Placement(transformation(extent={{50,-20},{70,0}})));
+  Buildings.Experimental.DHC.Networks.Distribution2PipePlugFlow disPlugFlow(
+    nCon=nLoa,
+    redeclare final package Medium = Medium1,
+    allowFlowReversal=false,
+    mDis_flow_nominal=m_flow_nominal,
+    mCon_flow_nominal=mCon_flow_nominal,
+    mEnd_flow_nominal=m_flow_nominal,
+    lDis=fill(6, nLoa),
+    lEnd=1) annotation (Placement(transformation(extent={{40,-80},{80,-60}})));
+  Buildings.Fluid.Movers.FlowControlled_dp pumPlugFlow(
+    redeclare package Medium = Medium1,
+    per(final motorCooledByFluid=false),
+    energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
+    m_flow_nominal=m_flow_nominal,
+    addPowerToMedium=false,
+    nominalValuesDefineDefaultPressureCurve=true,
+    use_inputFilter=false,
+    dp_nominal=dp_nominal)
+    annotation (Placement(transformation(extent={{0,-80},{20,-60}})));
+  Buildings.Fluid.MixingVolumes.MixingVolume volPlugFlow(
```

Comment:
Why are the volumes needed before the pumps in the distribution networks of the examples and not just supply/return boundary sources?

Reply:
The volumes are not needed, so I removed them.